### PR TITLE
change some images

### PR DESCRIPTION
**Pull request by rohitsingh7568**
change the commit and some images and add some log

### CODE REVIEW

File: src/components/sections/AssociatedPartners.tsx [hunk 1]
```diff
@@ -2,12 +2,12 @@ import Image from 'next/image'
...
 
 const partners = [
-  { name: "TechCorp", logo: "/images/partner-1.png" },
+  { name: "TechCorp", logo: "/images/manuj sir.png" },
   { name: "InnovateSoft", logo: "/images/partner-2.png" },
   { name: "GlobalMedia", logo: "/images/partner-3.jpeg" },
   { name: "DataDynamics", logo: "/images/partner-6.png" },
   { name: "CreativeEdge", logo: "/images/partner-5.png" },
-  { name: "MarketPro", logo: "/images/partner-7.png" },
+  { name: "MarketPro", logo: "/images/mishi media solution.png" },
```
Comment (Priyansh61) [inline src/components/sections/AssociatedPartners.tsx]: Add names relevant to the company logos

File: src/components/sections/GetInTouch.tsx [hunk 2]
```diff
@@ -19,7 +24,7 @@ export default function CTA() {
           Are You Ready? Let's Get to Work!
         </h2>
         <Link
-          href="#"
+          href="https://whatsapp.com/channel/0029VavEHHh0Qealmt5Vbd0o"
```
Comment (Priyansh61) [inline src/components/sections/GetInTouch.tsx]: Open a new tab use `target = "_blank"` helps to open in a new tab

File: src/components/sections/Services.tsx [hunk 3]
```diff
@@ -34,6 +34,16 @@ const services = [
...
+  },
+  {
+    title: "Graphic Designing",
+    description: "Graphic designing is the art of creating visual content to communicate messages effectively. It combines creativity, typography, color, and imagery to design everything from logos and branding to digital media and print materials, enhancing visual appeal and functionality.",
+    icon: Palette
+  },
+  {
+    title: "Video Production",
+    description: "Video production is the process of creating video content from start to finish, including planning, filming, and editing. It involves crafting visually engaging stories to communicate messages, entertain, or promote ideas effectively.",
+    icon: Video
```
Comment (Priyansh61) [inline src/components/sections/Services.tsx]: Align the last line in the center.

File: src/components/sections/Testemonial.tsx
Comment (Priyansh61) [inline src/components/sections/Testemonial.tsx:1]: Compress the images and use same scale size like 120*120

File: src/components/layout/Footer.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ export default function Footer() {
           <div className="col-span-1">
             <Link href="/" className="inline-block mb-6">
               <img
-                src="/images/logo.jpeg" // Replace with your logo path
+                src="/images/logo.svg" // Replace with your logo path
```
Comment (Priyansh61) [inline src/components/layout/Footer.tsx]: Add the new without background logo here

**Review by Priyansh61 — CHANGES_REQUESTED** [review]

**Comment by rohitsingh7568** [comment]
changes some link

**Pull request merged**